src/scikit_build_cli/commands/metadata.py

```diff
@@ -10,6 +10,7 @@ def __dir__() -> list[str]:
 
 
 @click.command()
+@click.help_option("--help", "-h")
 @click.pass_context
 def metadata(ctx: click.Context) -> None:  # noqa: ARG001
```

Comment:
I was thinking, this might make more sense to be `dist_info` and rename the dynamica_metadata to `metadata`? Are there any metadata outside of `dist-info` that might make it confusing?